### ISSUE TITLE
Toolbar orientation changes to Horizontal with Minimal UI module

### ISSUE THREAD

**Issue by ravingdragoon** [ardittristan/VTTButtonOverflow#8]
When using the Minimal UI module with the latest Button Overflow module, the issue did not occur previously, the primary left toolbar becomes horizontal, with the first button of the secondary toolbar covering the second button of the primary toolbar.

![image](https://user-images.githubusercontent.com/72875320/147581094-ef72d0cb-240e-4dd8-b3a1-a559c9e6ff9b.png)

```Versions:
  Foundry: 9.238
  System: dnd5e version 1.5.6
User:
  Role: GAMEMASTER
Settings:
  Disable_Canvas: Disabled
  Max_FPS: 60
  Token_Drag_Vision: Disabled
  Performance_Mode: 3
  Token_Vision_Animation: Enabled
  Light_Source_Animation: Enabled
  Zoomed_Texture_Antialiasing: Enabled
  Animate_Roll_Tables: Disabled
Scene:
  Walls: 709
  Lights: 2
  Tokens: 19
  Tiles: 42
  Sounds: 11
  Drawings: 22
  Notes: 11
  Dimensions: 4800 x 6600
  Background: 4970 x 6790
  Foreground: 1190 x 975
WebGL:
  Context: WebGL2RenderingContext
  GL_Vendor: WebKit
  Renderer: WebKit WebGL
  Unmasked_Renderer: ANGLE (NVIDIA, NVIDIA GeForce GTX 1080 Ti Direct3D11 vs_5_0 ps_5_0, D3D11-30.0.14.9729)
  WebGL_Version: WebGL 2.0 (OpenGL ES 3.0 Chromium)
  MAX_TEXTURE_SIZE: 16384
  MAX_RENDERBUFFER: 16384
Database:
  Actors: 78
  Items: 173
  Scenes: 3
  Journals: 793
  Tables: 95
  Chat: 231
  Macros: 398
Data_Sizes:
  Actors: 9.94 MB
  Items: 771.75 KB
  Scenes: 669.34 KB (2 Base64 Detected)
  Journals: 5.76 MB
  Tables: 1.49 MB
  Chat: 341.92 KB (2 Base64 Detected)
  Macros: 864.66 KB
Browser:
  Platform: Win32
  Vendor: Google Inc.
  Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36
Memory:
  Heap_Limit: 4 GB
  Heap_Total: 267.84 MB
  Heap_Used: 171.98 MB
Modules:
  Total: 400
  Enabled: 14
Active_Modules:
  _chatcommands: Library: Chat Commands v1.4.0
  acelib: Ace library v1.3.0
  arbron-context-menus: Arbron’s Context Menu Library v0.2.2
  buttonoverflow: Button Overflow v1.0.0
  colorsettings: lib - Color Settings v2.8.1
  debug-report: Debug Report Generator v0.0.5
  find-the-culprit: Find the culprit v1.4.0
  lib-captions: libCaptions v0.1.4
  lib-changelogs: Changelogs & Conflicts v0.7.6
  lib-wrapper: libWrapper v1.11.0.1
  libruler: libRuler v0.2.0
  minimal-ui: Minimal UI v1.4.0
  socketlib: socketlib v1.0.9
  forge-vtt: The Forge v1.10
```

**Comment by ardittristan** [comment]
hm, what are the settings you have in minimalui? (there is a different issue I will fix next update with minimal ui, but I'm not able to reproduce the horizontal bar)

**Comment by ravingdragoon** [comment]
I have tried several different configurations, but here is the current one

![image](https://user-images.githubusercontent.com/72875320/147674919-3fb52592-64cd-42d7-a1f6-ca34410c07b9.png)

**Comment by ardittristan** [comment]
closed with 1.0.1
